modules/simple_forms_api/app/services/simple_forms_api/pdf_stamper.rb

```diff
@@ -260,5 +147,16 @@ def self.default_page_configuration
         { type: :new_page }
       ]
     end
+
+    def self.get_page_configuration(page, position)
+      config = [
+        { type: :new_page },
+        { type: :new_page },
+        { type: :new_page },
+        { type: :new_page }
+      ]
+      config[page] = { type: :text, position: }
+      config
```

Comment:
This could be rewritten with `.tap`:

```suggestion
      [
        { type: :new_page },
        { type: :new_page },
        { type: :new_page },
        { type: :new_page }
      ].tap do |config|
        config[page] = { type: :text, position: }
      end
```

Reply:
Ah yes! I need to get more used to `tap`.